Vagrantfile

```diff
@@ -64,40 +64,33 @@ Vagrant.configure(2) do |config|
   # WINDOWS WORKER (win server 2019)
   config.vm.define :winw1 do |winw1|
     winw1.vm.host_name = "winw1"
-    winw1.vm.box = "StefanScherer/windows_2019"  
-    winw1.vm.network :private_network, ip:"#{windows_node_ip}"
-    winw1.vm.synced_folder ".", "/vagrant", disabled:true
-    winw1.vm.synced_folder "./sync/shared", "C:/sync/shared"
-    winw1.vm.synced_folder "./sync/windows/", "C:/sync/windows/"
-    winw1.vm.synced_folder "./forked", "C:/forked/"
+    winw1.vm.box = "./windows-2019.box"
```

Comment:
replace with `FriedrichWilken/Windows_2019-sig-win-dev-tool`

Reply:
Also, make sure to use the latest version of that box. We are constantly updating it.